### ISSUE TITLE
Win11: NVDA not reading indexing (i.e. 1 of 3) of radio buttons in windows apps (i.e. File Explorer)

### ISSUE THREAD

**Issue by SMTestGH** [nvaccess/nvda#12761]
### Steps to reproduce:
1. Press Win+E to open file explorer
2. Press F6 to take focus to status bar
3. NVDA's focus lands on first control in status bar (radio button)

### Actual behavior:
NVDA reads the label, grouping, and state of radio button, but does not read indexing (i.e. 1 of 2):

status bar
View Modes  grouping
Details  radio button  checked  Displays information about each item in the window.  (Ctrl+Shift+6)

### Expected behavior:
NVDA should read the indexing of the radio buttons. NVDA reads indexing as expected in HTML radio buttons: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_type_radio
Narrator/JAWS work as expected.

#### NVDA installed/portable/running from source:
Installed/portable

#### NVDA version:
2021.1

#### Windows version:
Windows 11 Insider builds

#### Name and version of other software in use when reproducing the issue:
N/A

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No. 

#### If add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
Could you provide another set of steps to reproduce? It seems the UI for file explorer has changed as I don't encounter any radio buttons.

**Comment by michaelDCurran** [comment]
For an AT such as NVDA to announce indexing for radio buttons in UI Automation such as the ones in Explorer, the UIA provider (Explorer) needs to expose this info, specifically UIA_PositionInSetPropertyId and UIA_SizeOfSetPropertyId, which it does not currently do.
Note that Narrator also does not announce indexing of these particular radio buttons.
